PacketNet.sln

```diff
@@ -1,9 +1,9 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.27130.2010
+VisualStudioVersion = 15.0.27130.2027

```

Comment:
Can you split the changes to PacketNet.sln here out into another commit? The .gitignore change is ok, I can see how it could be helpful in case other packages get pulled in, but changes to the .sln are too wide of a scope for a single commit imo. Also VisualStudio is misspelled in your commit.

Reply:
Ok, I'll try to make more atomic commits with single changes. It is gone be tough with the utilization of naming conventions because they span across the whole project but I'll keep that change to be the last.